File: app/src/main/java/com/apppoweron/circularrevealbuttondemo/SecondPageFragment.java
package com.apppoweron.circularrevealbuttondemo;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class SecondPageFragment extends BaseFragment {

    public static SecondPageFragment newInstance() {
        return new SecondPageFragment();
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_second_page, container, false);
    }

}

File: app/src/main/kotlin/com/apppoweron/circularrevealbuttondemo/SecondPageFragment.kt
package com.apppoweron.circularrevealbuttondemo

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

class SecondPageFragment : BaseFragment() {

    companion object {
        fun newInstance(): SecondPageFragment {
            return SecondPageFragment()
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_second_page, container, false)
    }



}
